vars/lvVITester.groovy

```diff
@@ -0,0 +1,6 @@
+def call(testPath, lvVersion){
+   echo "Running VITester."
+   branchDirectory = WORKSPACE.substring(0, WORKSPACE.lastIndexOf("\\"))
```

Comment:
I don't understand the naming of this variable. This looks like the test tools directory, but with a name like `branchDirectory`, I can't be sure. Give this variable a more meaningful name.

Reply:
WORKSPACE is:
`C:\Users\<USERNAME>\.jenkins\jobs\<ORG>\jobs\<REPO>\branches\master\workspace`

branchDirectory is:
`C:\Users\<USERNAME>\.jenkins\jobs\<ORG>\jobs\<REPO>\branches\master\`

Tools directory is:
`C:\Users\<USERNAME>\.jenkins\jobs\<ORG>\jobs\<REPO>\branches\master\niveristand-custom-device-testing-tools`

I was playing with the idea of base directory or something else. But since it is the master branch directory, I named it that.

Reply:
That may be how your WORKSPACE is set up, but the default (at least with the version of the GitHub Branch Source Plugin on our server) is something like `C:\jenkins\workspace\oolkit-custom-device_master-SYU4HVHEAEJZA2XQXH5ADRUUNQQNPHLET37RVH3PNANU5XGQBXJA`.

So, when you strip the last part of the directory, it has nothing to do with a branch of anything. I'm fine with baseDirectory.